sql/src/Main.sk

```diff
@@ -203,7 +204,12 @@ untracked fun main(): void {
             "Globally unique identity of this write stream",
           ),
         )
-        .arg(Cli.StringArg("user").about("Name of the user")),
+        .arg(Cli.StringArg("user").about("Name of the user"))
+        .arg(
+          Cli.BoolArg("enable-rebuilds").about(
+            "Allow write-csv to apply rebuilds. Used wrongly, can lead to data loss.",
```

Comment:
This is a bit ominous/spooky but doesn't say how to avoid using it wrongly.  Would it be right to say something to the effect of "don't enable this server-side"?